scripts/scripts.js

```diff
@@ -355,6 +355,33 @@ function decorateH3(main) {
   }
 }
 
+/**
+ * Wraps images followed by links within a matching <a> tag.
+ * @param {Element} container The container element
+ */
+function decorateLinkedPictures(container) {
+  [...container.querySelectorAll('picture')]
+    .filter((picture) => {
+      const parent = picture.parentElement;
+      const link = parent.nextElementSibling?.querySelector('a[href]');
+      try {
+        return parent.childElementCount === 1 && link
+          && new URL(link.href).pathname === new URL(link.textContent).pathname;
```

Comment:
Wouldn't `link.href === link.textContent` also work?
Should we also check that `link.parentElement.childElementCount === 1` also?

Reply:
> Wouldn't link.href === link.textContent also work?

no it would not :) I am wrong. because if someone uses a `.live` or `.page` url the text content will remain unchanged, while the href would be relative.